.github/workflows/ci.yaml

```diff
@@ -26,9 +26,11 @@ on:
       - cryostat-v[0-9]+.[0-9]+
 
 env:
-  CI_OPERATOR_IMG: quay.io/cryostat/cryostat-operator
-  CI_BUNDLE_IMG: quay.io/cryostat/cryostat-operator-bundle
-  CI_SCORECARD_IMG: quay.io/cryostat/cryostat-operator-scorecard
+  CI_USER: cryostat+bot
+  CI_REGISTRY: quay.io/cryostat
+  CI_OPERATOR_IMG: ${CI_REGISTRY}/cryostat-operator
```

Comment:
Not sure if Github Action supports this syntax. By inspecting the above action, I am seeing:

```
> Run redhat-actions/push-to-registry@v2
  with:
    image: cryostat-operator
    tags: 2.4.0-dev latest
    registry: quay.io/andrewazores
    username: andrewazores+github_ci
    password: ***
    tls-verify: true
  env:
    CI_USER: andrewazores+github_ci
    CI_REGISTRY: quay.io/andrewazores
    CI_OPERATOR_IMG: ${CI_REGISTRY}/cryostat-operator
    CI_BUNDLE_IMG: ${CI_REGISTRY}/cryostat-operator-bundle
    CI_SCORECARD_IMG: ${CI_REGISTRY}/cryostat-operator-scorecard
    REPOSITORY: 
    REF: 
```

I suppose it keeps the `${CI_REGISTRY}` instead of replacing it with correct value?

Reply:
Hmmm... it worked on my fork, but the docs do indeed still say that this isn't supported. I can just inline the same value to each variable then.

Reply:
Sure nice! Thanks!